README.md

```diff
@@ -7,7 +7,7 @@
 </p>
 
 <p align="center">
-    Next-gen browser and mobile automation test framework for Node.js.
+    Next-gen browser and mobile automation test work for Node.js.
```

Comment:
Why this change?

Reply:
this was just to make some PR. forgot to remove. will remove